### PR TITLE
docs/installation/plugins

### PR DESCRIPTION
**Pull request by evantahler**
A page about Plugins! https://www-grouparoo-com-git-docs-plugins.grouparoo.vercel.app/docs/installation/plugins 

 Includes:
* Installing a Plugin
* Available Community Plugins (loaded live from NPM)
* Paid Plugins
  * How to install
  * ui-enterprise (more will be on a www page later)

### CODE REVIEW

File: pages/docs/installation/plugins.mdx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+The Grouparoo command line tool is used to add a plugin to your project via `grouparoo install [package]`. You can learn more about the `grouparoo install` command [here](/docs/cli/setup). So, if you wanted to install the `@grouparoo/mysql` plugin, you would run:
+
+```bash
+grouparoo install @grouparoo/mysql # install the latest version
+```
+
+That's it!
+
+Please ensure that the versions of your plugins match the version of `@grouparoo/core` that you are using **exactly**. If you need to install a specific version of a plugin to match `@grouparoo/core`, you can:
```
Comment (andyjih) [inline pages/docs/installation/plugins.mdx]: "Please ensure that the versions of your plugins match the version of `@grouparoo/core` that you are using **exactly**."
->
"Please ensure that the versions of your plugins match the version of `@grouparoo/core` that you are using **exactly** in your `package.json` file."

File: pages/docs/installation/plugins.mdx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+<PluginsList />
+
+## Paid Plugins
+
+While many Grouparoo plugins are available for free via an open-source license, some require a commercial license.
+
+<Alert variant="primary">
+  If you are interested in obtaining a license to our commercial plugins, please{" "}
+  <a href="/meet">contact us</a>.
```
Comment (andyjih) [inline pages/docs/installation/plugins.mdx]: Can you include some ref params for tracking?

File: pages/docs/installation/plugins.mdx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+</Alert>
+
+### Obtaining Paid Plugins
+
+To install a paid plugin, you will need to authenticate to the npm registry to download and install the new package. Grouparoo will provide you with a "npm access token" which will grant you access to the paid Grouparoo plugins you have purchased for your organization. For every computer that needs access to those plugins (developer machine, CI, build server, etc) you will need to provide that token to npm in one of the following ways:
+
+Use an `.npmrc` file in your Grouparoo project. This can be checked into your project and Docker images if kept private. The contents of the file will look like:
```
Comment (andyjih) [inline pages/docs/installation/plugins.mdx]: "Use an `.npmrc`file ..."
->
"Create a new `.npmrc` file... "

**Review by andyjih — CHANGES_REQUESTED** [review]
Overall looks great! Some tiny tweaks. :)

File: pages/docs/installation/plugins.mdx (hunk 2 above)
Comment (evantahler) [reply pages/docs/installation/plugins.mdx]: I added them, but what do you expect those URL params to do?  It doesn't look like we send them though the form?  We know that lots of folks block google analytics, and sometimes to next.js a route changes is not a page change that Google Analytics will pick up on...

**Comment by evantahler** [comment]
@andyjih updated!

**Review by andyjih — APPROVED** [review]

File: components/docs/pluginsList.tsx [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    npm: string;
+    homepage: string;
+    repository: string;
+    bugs: string;
+  };
+}
+
+const ignoredPlugins = ["@grouparoo/ui-components", "@grouparoo/client-web"];
+
+export default function PluginsList() {
```
Comment (seancdavis) [inline components/docs/pluginsList.tsx:21]: Clever approach!

**Review by seancdavis — APPROVED** [review]
Love it! I could see a world in which _Plugins_ becomes its own section in the docs and there is documentation on each plugin (including community-built plugins). But after giving this some thought, I feel like you put it in the right location for the time being.

**One potential addition** to this work would be mentioning where plugins fit into the overall installation process on [/docs/installation](https://www-grouparoo-com-git-docs-plugins.grouparoo.vercel.app/docs/installation).

**Comment by evantahler** [comment]
@seancdavis link added to `/docs/installation`!

File: components/docs/pluginsList.tsx (hunk 4 above)
Comment (evantahler) [reply components/docs/pluginsList.tsx:21]: Heh, I really wanted something fancier than a table, but I couldn't come up with a good idea without having access to any assets (icons) as well

**Pull request merged**